src/test/HelmetTest.js

```diff
@@ -24,7 +24,7 @@ describe("Helmet", () => {
         describe("title", () => {
             it("can update page title", () => {
                 ReactDOM.render(
-                    <Helmet title={"Test Title"} />,
+                    <Helmet title={"Test Title"} defaultTitle={"Fallback"} />,
```

Comment:
Sorry, nitpick, but if you could please put these props on new lines, like your other test case below, that would be great.  Otherwise, everything looks perfect.